alternator/ttl.cc

```diff
@@ -441,21 +447,35 @@ future<> expiration_service::run() {
             }
         }
         // FIXME: replace this silly 1-second sleep by something smarter.
-        co_await seastar::sleep(std::chrono::seconds(1));
+        try {
+            co_await seastar::sleep_abortable(std::chrono::seconds(1), _abort_source);
+        } catch(seastar::sleep_aborted&) {}
```

Comment:
Shouldn't you co_return there?

Reply:
Good catch.
The weirid thing that it worked like this! Perhaps I tested the abort code in a slightly different version and then broke it. I'll test again.

Reply:
Oh, actually I think my code is correct.
If the sleep is aborted, all it does it to make it shorter. But then we go around the loop again, and notice the abort source was triggered, and co_return there. There is no need to do is specially in the catch.

Reply:
style nitpick: missing space between `catch` and `(`